app/models/winning_bid.rb

```diff
@@ -0,0 +1,17 @@
+class WinningBid
```

Comment:
we seem to have quite a few non-AR models. I am fine with putting anything model-y in `app/models`

I made it a separate class bc I realized I was instantiating the rules factory a bunch and then finding winning bidder based on that